package.json

```diff
@@ -12,12 +12,13 @@
     "typecheck": "tsc --project ./tsconfig.noemit.json"
   },
   "dependencies": {
-    "next": "12.1.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "next": "^12.1.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
Ezek maradhatnak inkabb lockolva egy bizonyos verziora? (A `^` nelkul)

Reply:
persze, csak a windowsnál npm-el raktam fel automatikusan, azért került bele

Reply:
csináltam hozzá egy új commitot, ahol elvileg ezeket is javítottam